fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/bridge/api/complex/Status.java

```diff
@@ -21,18 +21,18 @@
  */
 public class Status {
 
-    public static final int STATUS_OK = 0x00;
+    public static final byte STATUS_OK = 0x00;
 
-    public static final int STATUS_ERR = 0xff;
+    public static final byte STATUS_ERR = (byte) 0xff;
```

Comment:
Since this is a byte now, we can omit the cast in `SimulationTraciRequest` line 56